src/qt/test/wallettests.cpp

```diff
@@ -170,6 +170,16 @@ void TestGUI(interfaces::Node& node)
     sendCoinsDialog.setModel(&walletModel);
     transactionView.setModel(&walletModel);
 
+    {
+        // Check balance in send dialog
```

Comment:
just noting this test won't catch the current behavior, that's ok for now

Reply:
Yup #17588.